ui/app/styles/core/tabs.scss

```diff
@@ -41,6 +41,10 @@
 
     + * {
       margin-top: 5em;
+
+      &.closer {
```

Comment:
Can you change this to `is-closer` to match the other modifier classes throughout the code base?